apps/data-feeds-config-generator/src/data-services/chainlink_feeds.spec.ts

```diff
@@ -133,8 +133,8 @@ describe('Tests for functions over Aggregated Data', async () => {
       });
     });
 
-    test('should work getting pair if in docs any of the base / quote fields are present', () => {
-      const testPair = ['tLINK', 'tAVAX'];
+    test('should log warning and return empty pair if there is inconsistent', () => {
+      const testPair = ['', ''];
```

Comment:
The function should return `null` to indicate clearly that it has received input it can't handle.